examples/unix/c99/z_pub.c

```diff
@@ -26,7 +26,7 @@ int main(int argc, char **argv) {
     const char *mode = "client";
     char *clocator = NULL;
     char *llocator = NULL;
-    int n = 10;
+    int n = 2147483647;  // max int value by default
```

Comment:
It's safer to use `int n = ~0;` in such a way we always have the guarantee `n` is `max(int)` regardless the actual size of `int`.